### DISCUSSION TITLE
[FEATURE REQUEST] Cached content is not supported in this package.

### DISCUSSION THREAD

**Discussion by yosun** [Uralstech/UGemini#17] [Feature Requests]
In your multimodal prompt example it seems that as the chat gets longer - closer to 1 M context window - without cached content, things will become very very slow? Why is cached content not supported?

` Cached content is not supported in this package.`

**Comment by Uralstech** [comment]
This is not a bug. UGemini does not yet support cached content. I've changed the issue to a feature request.

**Comment by Uralstech** [comment]
@yosun I've added cached content support in 3a5dc8e299c4db838b33151eb426d69fbe10a150. I'll put up a tag for it soon.

**Comment by Uralstech** [comment]
@yosun, as of UGemini v2.0.0-preview.5, I have tested and confirmed that all methods except for `cachedConents.patch` work. I have not tested `patch` and it probably does not work.

**Comment by Uralstech** [comment]
@yosun, as of UGemini v2.0.0, `cachedConents.patch` does work.
